application/src/messenger.h

```diff
@@ -18,22 +18,30 @@ class Messenger : public QObject
     };
     Q_ENUM(MessengerError)
 
+    Q_PROPERTY(QString userId READ userId WRITE setUserId NOTIFY userIdChanged)
```

Comment:
```suggestion
    Q_PROPERTY(QString userId READ userId NOTIFY userIdChanged)
```
не думаю что в Qml стоит разрешать менять id